src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,44 @@
+.watch {
+  @include square($clock-size);
+
+  &__second {
+    position: absolute;
+    left: 50%;
+    height: 50%;
+    width: 1em;
+    border-radius: 10px;
+    background-color: #5f98a8;
+    transform-origin: bottom center;
+    animation: duration 60s linear infinite;
+
+    &::before {
+      content: "";
+      position: absolute;
+      left: 50%;
+      top: 100%;
+      height: 2em;
+      width: 2em;
+      border-radius: 50%;
+      background-color: #5f98a8;
+      transform: translate(-50%, -50%);
+    }
+  }
+
+  &__minute {
+    position: absolute;
+    left: 50%;
+    height: 40%;
+    top: 10%;
+    width: 1em;
+    border-radius: 10px;
+    background-color: #c25d5d;
+    transform-origin: bottom center;
+    animation: duration 3600s linear infinite;
+  }

```

Comment:
Some styles are common with `second` class. Add common class for this style or create SASS extend.